resource/stylesheets/dl-image-toolbox.css

```diff
@@ -0,0 +1,27 @@
+.link-like {
+    color: #0088cc;
+    cursor: pointer;
+    text-decoration: none;
+}
+
+.model-summary-container {
+    font-size: 12px;
+    overflow-y: scroll;
+    width: 600px;
+    margin: 0 auto;
+    padding: 10px 20px;
+    border: 1px solid rgb(226, 226, 226);
+    height: 400px;
+}
+
+.several-controls .controls + .controls {
+    margin-top: 5px;
+}
+
+.mapping-editor__selector {
```

Comment:
Is it ok to rename this file and remove the other unused rules ?

Reply:
Yes !